src/viewProviders/resources.ts

```diff
@@ -226,21 +241,34 @@ export async function loadLocalResources(): Promise<ContainerTreeItem<LocalKafka
     "Local Kafka clusters discoverable at port `8082` are shown here.",
   );
 
-  const localClusters: LocalKafkaCluster[] = await getLocalKafkaClusters();
-  if (localClusters.length > 0) {
+  const localResources: LocalResourceGroup[] = await getLocalResources();
+  if (localResources.length > 0) {
     const connectedId = "local-container-connected";
     // XXX: if we don't adjust the ID, we'll see weird collapsibleState behavior
     localContainerItem.id = connectedId;
     // enable the "Stop Local Resources" action
     localContainerItem.contextValue = connectedId;
-
+    // unpack the local resources to more easily update the UI elements
+    const localKafkaClusters: LocalKafkaCluster[] = [];
+    const localSchemaRegistries: LocalSchemaRegistry[] = [];
+    localResources.forEach((group) => {
+      localKafkaClusters.push(...group.kafkaClusters);
+      if (group.schemaRegistry) {
+        localSchemaRegistries.push(group.schemaRegistry);
+      }
+    });
+    // update the UI based on whether or not we have local resources available
+    await Promise.all([
+      setContextValue(ContextValues.localKafkaClusterAvailable, localResources.length > 0),
```

Comment:
I ... know(?) that we can't have a local schema registry w/o at least one local kafka cluster, but I think this line would read better / symmetriclu if was spelled:

setContextValue(ContextValues.localKafkaClusterAvailable, localKafkaClusters.length > 0)

and not pivot off of localResources at all.

Reply:
Agreed, I think we'll need some better separation here once we move to direct connections where we could still have a saved "local" instance where the individual resources may not be reachable.